Source/WebKit/Shared/ApplePay/WebPaymentCoordinatorProxy.h

```diff
@@ -88,13 +79,17 @@ namespace WebKit {
 class PaymentSetupConfiguration;
 class PaymentSetupFeatures;
 
-class WebPaymentCoordinatorProxy
+class WebPaymentCoordinatorProxy final
     : public IPC::MessageReceiver
     , private IPC::MessageSender
-    , public PaymentAuthorizationPresenter::Client {
+    , public PaymentAuthorizationPresenter::Client
+    , public RefCounted<WebPaymentCoordinatorProxy> {
     WTF_MAKE_TZONE_ALLOCATED(WebPaymentCoordinatorProxy);
 public:
-    struct Client {
+    struct Client : public CanMakeWeakPtr<Client>, public CanMakeCheckedPtr<Client> {
+        WTF_MAKE_STRUCT_FAST_ALLOCATED;
+        WTF_STRUCT_OVERRIDE_DELETE_FOR_CHECKED_PTR(Client);
```

Comment:
Seems like `WebPageProxy::Internals` needs this too.

Reply:
Indeed, I forgot that. Will fix.